extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -173,6 +188,8 @@ oppia.directive('imageWithRegionsEditor', [
         // Convert to and from region area (which is stored as a fraction of
         // image width and height) and actual width and height
         var regionAreaFromCornerAndDimensions = function(x, y, width, height) {
+          height = Math.abs(height);
```

Comment:
Only while resizing the height and width computed can become if direction changes. Added a comment for the same in the resize region function